### PR TITLE
Fix formatting makefile cmds

### PR DESCRIPTION
**Pull request by ValarDragon**
make tools didn't install the relevant commands, so instead just use the single flow from golangci.
Also comments out linters without go1.18 support, and stops `tools-stamp` file from getting vendored

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -198,4 +198,5 @@ $RECYCLE.BIN/
 /artifacts/
 /.vscode/
 /scripts/local/
-/x/incentives/keeper/osmosis_testing/
\ No newline at end of file
+/x/incentives/keeper/osmosis_testing/
+tools-stamp
```
Comment (p0mvn) [inline .gitignore:202]: nit: missing new line

**Review by p0mvn — APPROVED** [review]
LGTM

**Review by alexanderbez — COMMENTED** [review]
Why not just update to go 1.18 in `go.mod` and uncomment the linters?

**Comment by ValarDragon** [comment]
Those linters don't work with go 1.18 installed, and we have to wait until they update / golangci updates for them to work again

**Comment by alexanderbez** [comment]
Ahh makes sense. Thankfully the Go ecosystem tooling moves quickly!

**Pull request merged**